### PR TITLE
scripts: add get_struct_size helper function

### PR DESCRIPTION
**Pull request by davidlunarg**
This function will be needed by vktrace in VulkanTools for adding pNext structures to trace files.

### CODE REVIEW

File: scripts/helper_file_generator.py [hunk 1]
```diff
@@ -884,9 +904,7 @@ def GenerateSafeStructSource(self):
             copy_construct_txt = construct_txt.replace(' (in_struct->', ' (src.')     # Exclude 'if' blocks from next line
             copy_construct_txt = copy_construct_txt.replace('(in_struct->', '(*src.') # Pass object to copy constructors
             copy_construct_txt = copy_construct_txt.replace('in_struct->', 'src.')    # Modify remaining struct refs for src object
-            copy_assign_txt = '    if (&src == this) return *this;\n\n' + destruct_txt + '\n' + copy_construct_init + copy_construct_txt + '\n    return *this;'
```
Comment (mark-lunarg) [inline scripts/helper_file_generator.py]: Hey @davidlunarg, it looks like this PR clobbers a recent change @chrisforbes made -- might wanna rebase and push again.

Comment (davidlunarg) [reply scripts/helper_file_generator.py]: Reinserted change from @chrisforbes, squashed it into my first commit and did a force push of my branch.

**Comment by davidlunarg** [comment]
I did another force push of the branch for this pull request. This latest force push adds a declaration of get_struct_size() to the vk_struct_size_helper.h header file.

**Comment by davidlunarg** [comment]
Cherry-picked to master

**Pull request closed without merging**